pkg/task/backup.go

```diff
@@ -250,6 +250,7 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 		if err2 != nil {
 			return err2
 		}
+		log.Warn("N0thing to backup, maybe connected to cluster for restoring")
```

Comment:
We can print the PD address here, hence our users can know which cluster it is.

BTW, why `N0thing`...?